src/libraries/System.Net.Quic/tests/FunctionalTests/MsQuicTests.cs

```diff
@@ -307,7 +307,7 @@ public async Task ConnectWithCertificateForDifferentName_Throws()
             await Assert.ThrowsAsync<AuthenticationException>(async () => await clientTask);
         }
 
-        [Theory]
+        [ConditionalTheory]
```

Comment:
Is this change intended?

Reply:
Yes, `Conditional` makes the test ignore the `SkipException`, otherwise it fails.
I assume the tests in CI might run without IPv6 support (only docker) so it didn't cause noise on your PR, but it was on my machine.

Reply:
I was wondering why `SkipException` sometimes fails and other times skips...., thanks.